service/src/main/java/uk/gov/hmcts/reform/fpl/service/MigrateCaseService.java

```diff
@@ -901,4 +904,48 @@ public void rollbackMigratedSkeletonArgumentList(CaseDetails caseDetails) {
         caseDataMap.put("skeletonArgumentList", skeletonArgumentList);
         caseDataMap.remove("skeletonArgumentListLA");
     }
+
+    public Map<String, Object> migrateCorrespondenceDocuments(CaseData caseData) {
+        List<Element<ManagedDocument>> correspondenceDocList =
+            Stream.of(caseData.getCorrespondenceDocuments(),
+                    caseData.getCorrespondenceDocumentsLA(),
+                    caseData.getCorrespondenceDocumentsSolicitor())
+                .flatMap(Collection::stream)
+                .filter(bundleElement -> !bundleElement.getValue().isConfidentialDocument())
+                .map(bundleElement ->
+                    element(bundleElement.getId(),
+                        ManagedDocument.builder().document(bundleElement.getValue().getDocument()).build()))
+                .collect(toList());
+
+        List<Element<ManagedDocument>> correspondenceDocListLA =
+            Stream.of(caseData.getCorrespondenceDocumentsLA(),
+                    caseData.getCorrespondenceDocumentsSolicitor())
```

Comment:
is there any case that element from`correspondenceDocumentsSolicitor` is uploaded by LA?

Reply:
Think no, LA solicitor / barrister will go to "manageDocumentsLA" and upload fromcorrespondenceDocumentsLA

Resp / child solicitor goes to "manageDocuments" and upload correspondenceDocumentsSolicitor

Reply:
so we don't need `getCorrespondenceDocumentsSolicitor` here?

Reply:
I think LA level in new Mange Doc flow is for confidential document, right? And CTSC level is for those confidential uploaded by HMCTS staff? 
I am not sure if there is any confidential document uploaded by solicitor or not.

Reply:
Yes, `xxxListLA` is for confidential document limited to LA and `xxxListCTSC`is for those confidential uploaded by HMCTS staff.

The confidential checkbox will not be displayed to solicitor. So documents under `correspondenceDocumentsSolicitor` should be non-confidential.

Reply:
I was just thinking if solicitor uploaded a confidential documents, the documents should be still accessible by them. So , it should be kept under non-confidential collecition.